docs/reference/calculate_ranks.html

```diff
@@ -81,7 +81,7 @@
       <ul class="nav navbar-nav">
         <li>
```

Comment:
I ask myself why calculate_ranks.html exists here since this function isn't exported anymore.

Reply:
{pkgdown} renders every help page regardless whether a function is exported or not. We could either delete the documentation as internal functions don't need documentation or simply keep everything as it is, which is what I prefer